torch/onnx/_internal/_beartype.py

```diff
@@ -45,18 +45,12 @@ def _create_beartype_decorator(
         return _no_op_decorator
     if _beartype_lib is None:
         # If the beartype library is not installed, return a no-op decorator
-        if runtime_check_state == _exporter_states.RuntimeTypeCheckState.ERRORS:
-            warnings.warn(
-                "TORCH_ONNX_EXPERIMENTAL_RUNTIME_TYPE_CHECK is set to 'ERRORS', "
-                "but the beartype library is not installed. "
-                "Install beartype with `pip install beartype` to enable runtime type checking."
-            )
         return _no_op_decorator
 
     assert isinstance(_beartype_lib, ModuleType)
 
     if runtime_check_state == _exporter_states.RuntimeTypeCheckState.ERRORS:
-        # Enable runtime type checking which errors on any type hint violation.
+        # Enable runtime type checking with errors on any type hint violation.
```

Comment:
verb to noun for errors 😆

Reply:
Ha! I should keep the original. For some reason my brain forgot that error can be a verb